virtual/vbox/vbox_test.go

```diff
@@ -83,3 +86,57 @@ func TestVmBase(t *testing.T) {
 		t.Fatalf("expected virtual machine to have been removed")
 	}
 }
+
+func TestSetRAMandCPU(t *testing.T) {
+	tst.SkipCI(t)
+	ramre := regexp.MustCompile(`memory=[1-9][0-9]*`)
+	cpure := regexp.MustCompile(`cpus=[0-9]*`)
+	memorysize:=1024
+	ctx := context.Background()
+	cs := "d41d8cd98f00b204e9800998ecf8427e"
+	vm := vbox.NewVMWithSum("haaukins.ova", "haaukins", cs)
+	vm.Create(ctx)
+	vm.Snapshot("test_haaukins")
+
+	linkedCloneVM,err := vm.LinkedClone(ctx,"test_haaukins",vbox.SetRAM(uint(memorysize)),vbox.SetCPU(uint(2)))
+	if err != nil {
+		t.Fatalf("Linked clone could not created %s ", err)
+	}
+
```

Comment:
Be careful with closing the VMs. For instance, if the next function call fails, the `linkedCloneVm` will never close, resulting in a dangling VM on the system that is running the test. So, I'd recommend using `defer` statements instead. That will ensure that a certain function will be called, regardless of test failure.

Reply:
👍🏻 Thank you for the feedback :) Updating...